.travis.yml

```
@@ -12,3 +19,13 @@ before_script:
 script:
   - yarn test:coverage
   - yarn codecov
+  - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo "$TRAVIS_BRANCH"; else echo "$TRAVIS_PULL_REQUEST_BRANCH"; fi)
+  - if [ "$BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then ./release.sh compare_and_release; fi
+
+deploy:
+  provider: npm
+  email: '$NPM_EMAIL'
+  api_key: '$NPM_API_TOKEN'
```

Let's make it `NPM_TOKEN` for consistency.

```suggestion
  api_key: '$NPM_TOKEN'
```